docs/debugging-tips.adoc

```diff
@@ -137,3 +137,29 @@ sudo tcpdump tcp port 443 -w upload.pcap
 ```
 
 Fetch both of these down. In wireshark preferences set ssl.keylog_file to point to premaster.txt. If your https traffic not on port 443, then add the port to http.ssl.port. Now open upload.pcap.
+
+== Serve repo generated by aktualizr-repo
```

Comment:
It could be made clearer by giving an example of `aktualizr-repo` invocation.

For example:

```
aktualizr can be tested against a dummy repository containing fake images

First, generate a repository using `aktualizr-repo`:

    aktualizr-repo generate --path repo_dir
    {other commands?}

Then, serve the generated directory using a web server. Here is an example configuration for nginx:

{nginx config}

{here, maybe explain how to use it with aktualizr or tests}
```